src/components/SearchResults/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import Header from '../Header'
+import ResultsTable from './ResultsTable'
+import './index.css'
+import Spin from '../../lib/spinner'
+
+const onLoadSearch = async searched => {
+  const res = await fetch(
+    `${process.env.REACT_APP_API}/donors/search?name=${searched}`,
+    {
+      method: 'GET',
+      headers: { 'Content-Type': 'application/json' }
+    }
+  )
+  const data = await res.json()
+  return data
+}
+
+function SearchResults() {
+  const searched = useParams().name
+  const [searchResults, setSearchResults] = useState('')
+
+  useEffect(() => {
+    onLoadSearch(searched).then(data => {
+      setSearchResults(data)
+    })
+  }, [searched])
+
+  if (searchResults) {
+    return (
+      <div>
+        {console.log(searchResults)}
+        <Header>
+          <Header.Top>
+            <Header.Content>
+              <div className="totaldonationamt">
+                Search Results for "{searched}"
+              </div>
+              <div className="keystatslabel">
+                {message(searchResults.length)}
+              </div>
+            </Header.Content>
+          </Header.Top>
+        </Header>
+        <ResultsTable data={searchResults} />
```

Comment:
Can add a condition that checks `searchResults.length` is greater than `0` before it shows `ResultsTable`. Right now if there are no results, it's an empty table with the header still visible.

![image](https://user-images.githubusercontent.com/3888250/77565551-58938480-6eff-11ea-85a3-4dc3d0c5c9ba.png)